eclair-core/src/main/scala/fr/acinq/eclair/balance/BalanceActor.scala

```diff
@@ -75,33 +75,35 @@ private class BalanceActor(context: ActorContext[Command],
           )
           filteredByStatus.foreach {
             case (status, filteredUtxos) =>
-              val amount = filteredUtxos.map(_.amount.toDouble).sum
+              val amount = filteredUtxos.map(_.amount).fold(MilliBtc(0))(_ + _)
```

Comment:
@sstone Is there a reason why `scala.compat.Btc` (and others) does not implement `Numeric[Btc]`? We can't use `sum` because of that.

Reply:
I'll look into it, meanwhile You can use `sum` on `Satoshi` because we provide an implicit conversion to `Numeric[Satoshi]`.

Reply:
Nice, done in 56816a081.